cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -468,10 +468,11 @@ func (c *revidCameraClient) voltage(ctx *broadcastContext) (float64, error) {
 func (c *revidCameraClient) alarmVoltage(ctx *broadcastContext) (float64, error) {
 	// Get AlarmVoltage variable; if the voltage is above this we expect the controller to be on.
 	// If the voltage is below this, we expect the controller to be off.
-	alarmVoltageVar, err := model.GetVariable(context.Background(), ctx.store, ctx.cfg.SKey, "AlarmVoltage")
+	alarmVoltageVar, err := model.GetVariable(context.Background(), ctx.store, ctx.cfg.SKey, fmt.Sprintf("%012x", ctx.cfg.ControllerMAC)+".AlarmVoltage")
```

Comment:
Could you just ParseInt ?

Reply:
Yeah sorry mi have dumb moment. What about ```strconv.Itoa```

Reply:
I could use strconv.FormatInt(value, 16) apparently for hex, but to pad with 0s what I have is the simplest way, which is what model.Device.Hex() uses but I can't use that here since I don't have access to the device.

Reply:
hhhmm, I almost think we should get the device some how. I just think that's a better pattern that doing this.

Reply:
Why? Is there a reason not to trust the configured controller MAC address?

Reply:
No, I just mean that it's good to follow the same pattern of using the device and Hex method; it seems this is how we get variables everywhere else.

Reply:
What about this?:

```
controllerMACHex := (&model.Device{Mac: ctx.cfg.ControllerMAC}).Hex()
alarmVoltageVar, err := model.GetVariable(context.Background(), ctx.store, ctx.cfg.SKey, controllerMACHex+".AlarmVoltage")
```

I'm trying to avoid a network request for the device when all we need is the MAC.

Btw SPrintf for the Mac is also used in OceanManager.SetupSecondary so it's not the only place that it's done this way. And it's not really that complex, it's just another representation of the same value.

Reply:
The reason I create `controllerMACHex` is because it's used twice.